packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1363,7 +1365,7 @@ export interface CustomCallControlButtonCallbackArgs {
 }
 
 // @beta
-export type CustomCallControlButtonPlacement = 'first' | 'last' | 'afterCameraButton' | 'afterEndCallButton' | 'afterMicrophoneButton' | 'afterDevicesButton' | 'afterParticipantsButton' | 'afterScreenShareButton';
+export type CustomCallControlButtonPlacement = 'mainBar' | 'sideBar' | 'overflowMenu';
```

Comment:
I think `Bar` is confusing here, `sideBar` would imply inside the panel that appears from the side but buttons placed here could be independant of the side bar.
Perhaps `primary | secondary | primaryOverflow` ? (maybe primaryControls or primarySpace .. very open to suggestions)

Reply:
We could even go with:
`primaryControlBar | 'secondaryControlBar | 'overFlowControlBar`
or something a bit more basic
`primary | 'secondary | 'overflow`

Reply:
Decided to go with 
`primary | secondary | overflow`

Please comment if there is an issue with what I used :)